flocker/node/agents/ebs.py

```diff
@@ -107,6 +115,68 @@ def _wait_for_volume(expected_volume,
             )
 
 
+def _check_blockdevice_size(device, size):
+    """
+    Helper function to check if the size of block device with given
+    suffix matches the input size.
+
+    :param unicode device: Name of the block device to check for size.
+    :param int size: Size, in SI metric bytes, of device we are interested in.
+
+    :returns True if a block device with given name has given size.
+        False otherwise.
+    """
+    device_name = b"/dev/" + device
```

Comment:
Concatenating `bytes` and `unicode` here.  Should be something like `b"/dev/" + device.encode("ascii")` instead.